lib/index.js

```diff
@@ -88,6 +70,19 @@ export default class extends Component {
     })
   }
 
+  static getDerivedStateFromProps(nextProps, prevState) {
+    if (nextProps.value !== prevState.value) {
+      return {value: prevState.value}
+    }
+    else return null;
+  };
```

Comment:
It could be better like this:
```javascirpt
 return nextProps.value !== prevState.value ? { value: prevState.value } : null;
```

or 
```javascript
 if (nextProps.value !== prevState.value) {
      return {value: prevState.value}
 } else {
      return null;
 }
```

And code style isn't right around line 75;